.taskcluster.yml

```diff
@@ -25,6 +25,9 @@ tasks:
       $if: 'tasks_for == "github-pull-request"'
       then: ${event.pull_request.head.repo.html_url}
       else: ${event.repository.html_url}
+
+    mozilla_central: gecko-dev
+    mozilla_central_output: /tmp/mozilla_central_output
```

Comment:
Let's hardcode the directories in the task itself, it's not worth defining them separately far from the task

Reply:
How should I define a `let` in a single task?

Reply:
You can just completely avoid defining the variable and just hardcode the actual values (`gecko-dev` and `/tmp/mozilla_central_output`) where they are used